agave_app/renderDialog.cpp

```diff
@@ -791,6 +798,92 @@ RenderDialog::getFullSavePath()
   return saveFilePath;
 }
 
+void
+RenderDialog::onFrameDone(QImage image)
```

Comment:
most of the body of this function used to be in onRenderRequestProcessed, where we would try to figure out if the frame was completed or not.  With this message handler, it is more definitive.